fhir-server/liberty-config/config/default/fhir-server-config-db2.json

```diff
@@ -117,30 +117,83 @@
             }
         },
         "bulkdata": {
-            "bulkDataBatchJobIdEncryptionKey": "change-password",
-            "applicationName": "fhir-bulkimportexport-webapp",
-            "moduleName": "fhir-bulkimportexport.war",
-            "jobParameters": {
-                "cos.bucket.name": "fhirbulkdata",
-                "cos.location": "us",
-                "cos.endpoint.internal": "https://minio:9000",
-                "cos.endpoint.external": "https://minio:9000",
-                "cos.credential.ibm": "N",
-                "cos.api.key": "minio",
-                "cos.srvinst.id": "change-password"
+            "enabled": true,
+            "core": {
+                "api": {
+                    "url": "https://localhost:9443/ibm/api/batch",
+                    "user": "fhiradmin",
+                    "password": "change-password",
+                    "truststore": "resources/security/fhirTrustStore.p12",
+                    "truststore-password": "change-password", 
+                    "trust-all": true
+                },
+                "cos" : { 
+                    "max-resources": 200000,
+                    "min-size": 10485760,
+                    "max-size": 209715200,
+                    "use-server-truststore": true
+                },
+                "page-size": 100,
+                "batch-id-encryption-key": "change-password",
+                "max-partitions": 3, 
+                "max-inputs": 5,
+                "systemExportImpl": "none"
             },
-            "implementation_type": "cos",
-            "batch-uri": "https://localhost:9443/ibm/api/batch/jobinstances",
-            "batch-user": "fhiradmin",
-            "batch-user-password": "change-password",
-            "batch-truststore": "resources/security/fhirTrustStore.p12",
-            "batch-truststore-password": "change-password",
-            "isExportPublic": true,
-            "cosFileMaxResources": 200000,
-            "cosFileMaxSize": 209715200,
-            "validBaseUrls": [],
-            "useFhirServerTrustStore": true,
-            "enableParquet": false
+            "storageProviders": {
+                "default" : {
+                    "type": "file",
+                    "_type": "ibm-cos|aws-s3|file|https",
+                    "valid-base-urls": [],
```

Comment:
previously we've stuck to CamelCase for config props, but I think I'm OK with consistently using kabob-case on these culk properties...anyone disagree?

Reply:
flipping to camelcase is quick  change.   just let me know